notebooks/3.jump-analysis/3.jump_analysis.sh

```diff
@@ -8,4 +8,3 @@ jupyter nbconvert --to python --output-dir=nbconverted/ *.ipynb
 
 # run the scripts
 python nbconverted/3.jump_analysis.py
-python nbconverted/3.1.overlapping_compounds.py
```

Comment:
Is this no longer a script you need to run? Just confirming this was not included in error.

Reply:
Is this related to removing the old code?

Reply:
Correct, I removed the script but I forgot to remove the command in the .sh file